src/EditorFeatures/Core/Implementation/InlineRename/CommandHandlers/RenameCommandHandler.cs

```diff
@@ -77,17 +78,22 @@ private void HandlePossibleTypingCommand(CommandArgs args, Action nextHandler, A
                 // It's in a read-only area, so let's commit the rename and then let the character go
                 // through
 
-                CommitIfActiveAndCallNextHandler(args, nextHandler);
+                TryCommitIfActiveAndCallNextHandlerIfCommitted(args, nextHandler);
             }
         }
 
-        private void CommitIfActiveAndCallNextHandler(CommandArgs args, Action nextHandler)
+        private void TryCommitIfActiveAndCallNextHandlerIfCommitted(CommandArgs args, Action nextHandler)
```

Comment:
Hmm, it's not the typical "Try to do this thing" pattern, it's "Call 'TryCommit' and then proceed."

Maybe I'll change it to ```RequestCommitIfActiveAndCallNextHandlerIfNoActiveSession``` or something.